### ISSUE TITLE
bricking

### ISSUE THREAD

**Issue by nepoladka** [J0xna/Kernel-Overlay-Hider#1]
hi

on win10 21h2 19044.2486 after hiding the window, sometimes the system may get bricked when interacting with the taskbar

did you face similar problem? any ideas how to find the problem and fix it?

**Comment by J0xna** [comment]
Yes, literally typed it in the source, I won't provide any fixes :)

**Comment by nepoladka** [comment]
i missed it, sorry.

thanks for the help anyway!
